app/backend/prepdocslib/searchmanager.py

```diff
@@ -65,140 +67,190 @@ def __init__(
-                        analyzer_name="keyword",
-                    )
-                ),
-                SearchableField(
-                    name="content",
-                    type="Edm.String",
-                    analyzer_name=self.search_analyzer_name,
-                ),
-                SearchField(
-                    name="embedding",
-                    type=SearchFieldDataType.Collection(SearchFieldDataType.Single),
-                    hidden=False,
-                    searchable=True,
-                    filterable=False,
-                    sortable=False,
-                    facetable=False,
-                    vector_search_dimensions=self.embedding_dimensions,
-                    vector_search_profile_name="embedding_config",
-                ),
-                SimpleField(name="category", type="Edm.String", filterable=True, facetable=True),
-                SimpleField(
-                    name="sourcepage",
-                    type="Edm.String",
-                    filterable=True,
-                    facetable=True,
-                ),
-                SimpleField(
-                    name="sourcefile",
-                    type="Edm.String",
-                    filterable=True,
-                    facetable=True,
-                ),
-                SimpleField(
-                    name="storageUrl",
-                    type="Edm.String",
-                    filterable=True,
-                    facetable=False,
-                ),
-            ]
-            if self.use_acls:
-                fields.append(
-                    SimpleField(
-                        name="oids",
-                        type=SearchFieldDataType.Collection(SearchFieldDataType.String),
-                        filterable=True,
-                    )
-                )
-                fields.append(
-                    SimpleField(
-                        name="groups",
-                        type=SearchFieldDataType.Collection(SearchFieldDataType.String),
-                        filterable=True,
-                    )
-                )
-            if self.use_int_vectorization:
-                fields.append(SearchableField(name="parent_id", type="Edm.String", filterable=True))
-            if self.search_images:
-                fields.append(
+                        analyzer_name=self.search_analyzer_name,
+                    ),
                     SearchField(
-                        name="imageEmbedding",
+                        name="embedding",
                         type=SearchFieldDataType.Collection(SearchFieldDataType.Single),
                         hidden=False,
                         searchable=True,
                         filterable=False,
                         sortable=False,
                         facetable=False,
-                        vector_search_dimensions=1024,
+                        vector_search_dimensions=self.embedding_dimensions,
```

Comment:
Git is not diffing this field correctly, the imageEmbedding field is still 1024.